src/robots/yamcs_TMTC.py

```diff
@@ -52,6 +61,10 @@ def _command_callback(self, command:CommandHistory):
             self._drive_robot_straight(arguments["linear_velocity"], arguments["distance"]) 
         elif name == self._yamcs_conf["commands"]["drive_turn"]:
             self._drive_robot_turn(arguments["angular_velocity"], arguments["angle"])
+        elif name == self._yamcs_conf["commands"]["camera_capture_high"]:
+            self._camera_handler.transmit_camera_view(self.IMAGES_ONCOMMAND, "high")
```

Comment:
instead of self.IMAGES_ONCOMMAND you can use CameraViewTransmitHandler.IMAGES_ONCOMMAND. more explicit and avoids defining these magics in two places

(renamed to BUCKET_IMAGES_ONCOMMAND. see comment in CameraViewTransmitHandler class)